docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| `iPAddress`                 | Y             | MUST contain the IPv4 or IPv6 address that the CA has confirmed the Applicant controls or has been granted the right to use through a method specified in [Section 3.2.2.5](#3225-authentication-for-an-ip-address). MUST NOT contain a Reserved IP Address. |
+| `registeredID`              | N             | -              |
+
+**Note**: If the `subject` field of the certificate is an empty SEQUENCE, this extension MUST be marked critical, as specified in [RFC 5280, Section 4.2.1.6](https://tools.ietf.org/html/rfc5280#section-4.2.1.6).
+
+#### 7.1.2.7 OCSP Responder Certificate Profile
+
+If the Issuing CA does not directly sign OCSP responses, it MAY make use of an OCSP Authorized Responder. The Issuing CA of the Responder MUST be the same as the Issuing CA for the Certificates it provides responses for.
+
+| __Field__                      | __Description__ |
+| ---                            | ------          |
+| `tbsCertificate`               | |
+| \ \ \ \ `version`              | MUST be v3(2) |
+| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) containing at least 64 bits of output from a CSPRNG. |
+| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+| \ \ \ \ `issuer`               | MUST be byte-for-byte identical to the `subject` field of the Issuing CA. See [Section 7.1.4.1](#7141-name-encoding) |
+| \ \ \ \ `validity`             | __**TBD**__ |
+| \ \ \ \ `subject`              | See [Section 7.1.2.8.1](#71281-ca-certificate-naming) |
+| \ \ \ \ `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
+| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
+| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
+| \ \ \ \ `extensions`           | See [Section 7.1.2.5.1](#71251-tls-subordinate-ca-extensions) |
+| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.7.1 OCSP Responder Extensions
+
+| __Extension__                     | __Presence__ | __Critical__ | __Description__ |
+| ----                              | -            | -            | ----- |
+| `authorityKeyIdentifier`          | MUST         | N            | See [Section 7.1.2.9.2](#71292-authority-key-identifier) |
+| `basicConstraints`                | MUST         | Y            | See [Section 7.1.2.7.3](#71273-basic-constraints) |
+| `extKeyUsage`                     | MUST         | -            | See [Section 7.1.2.7.4](#71274-extended-key-usage) |
+| `id-pkix-ocsp-nocheck`            | MUST         | N            | See [Section 7.1.2.7.5](#71275-id-pkix-ocsp-nocheck) |
+| `keyUsage`                        | MUST         | Y            | See [Section 7.1.2.7.6](#71276-key-usage) |
+| `certificatePolicies`             | MUST NOT     | N            | - |
```

Comment:
The purpose of including `certificatePolicies` is to allow a verifying client application to verify that the certificate complies with those policies.

No such hook is afforded to OCSP verification algorithm; it's an unnecessary superfluous input. The purpose of these profiles is to clarify how such logical information flows, to remove unnecessary inputs that might otherwise harm interoperability.

Could you explain the justification for including the certificate policy for relying parties?

Reply:
The OCSP responder certificate is part of the CA's infrastructure and it would be used by Relying Parties to verify the status of certificates. It makes sense for me to include a policy identifier to point to a CPS indicating how this certificate was issued, what protections exist around this certificate, and so on.

I don't have strong feelings about it. If the SCWG wants to MUST NOT this extension, it should be done with a future effective date.

Reply:
I'm not sure, but I think your latest comment just restates what I was highlighting is problematic: "Because we want to" isn't a good justification, especially when it explicitly has no bearing for Relying Parties for OCSP responders. The only way I can make sense of "used by Relying Parties" is if you're suggesting that individual users should personally manually inspect the OCSP responder certificate, which certainly would seem problematic.

>  If the SCWG wants to MUST NOT this extension, it should be done with a future effective date.

I understand that's your default position on any/every change, and I think we've captured on enough ballots that I find such statements vexatious and problematic. While I do think, as a whole, there will inevitably need to be some transition (even if it's 1:1 matching our existing requirements, inevitably Some CA Somewhere will argue that they Didn't Know), but if you believe that this is anything more difficult than something that can and should be done immediately, my hope is that you very precisely explain.

Given that the baseline expectation of at least Chrome is that every CA should be able to immediately change certificate profiles **or** be able to concretely articulate why that would be problematic, my hope is you could elaborate specifically if there's something you believe would cause compatability issues here. Otherwise, I don't think adding a specific transition date for just this field is appropriate or something we'll support.

Reply:
Yes, I suggested that individual users, including security researchers, analysts, supervisory bodies, personally manually inspect this information.

Speaking for HARICA, we can update our profiles and issue new OCSP responder certificates quite fast. Updating profiles and issuing new OCSP responder certificates from offline CAs require more time and preparation. The COVID-19 travel and lock-down restrictions need to be considered. Hopefully when this ballot is introduced, most of the pandemic will be under control. However, this "MUST NOT" should not be assumed to be an "immediate" change.

Reply:
Like Dimitris, I'm not sure I follow the rationale for prohibiting certificatePolicies. Client implementations are free to ignore policy chaining if they so desire by specifying the appropriate inputs to the path-building algorithm in section 6 of 5280. Likewise, other client implementations may desire to enforce a particular policy (or set of policies) for delegated OCSP responder certificate verification. Prohibiting the certificatePolicies extension would preclude a client from ever doing the latter.

Reply:
> Yes, I suggested that individual users, including security researchers, analysts, supervisory bodies, personally manually inspect this information.

That’s not what `certificatePolicies` are for, though, and the idea that folks are manually inspecting OCSP responses for policy information seems troubling.

> Likewise, other client implementations may desire to enforce a particular policy (or set of policies) for delegated OCSP responder certificate verification.

So, the BRs absolutely won’t require this, because this is superfluous information for the BRs: all responders follow a single policy (the BRs). The use of any other policy identifier is for the purposes of combining trust frameworks, and that’s fundamentally something we’re going to object to.

However, your “what if” is also not supported by the implementations, in that no significant PKIX implementation that supports OCSP verification allows passing in an initial policy set, so this whole hypothetical is moot. It’s not even part of the 6960 validation process.

I get the desire for “what if”, and we can re-evaluate changing the policy with concrete use cases, but this demonstrably has no technical purpose for the validation. I get the “it’s nice”, and I get “we need a phase in”, but this is something whose inclusion is fundamentally problematic, has no technical value, and has no compatibility risk today for forbidding.

Reply:
This is now set for a phase-in to avoid hassle.